model/testing/src/icon4py/model/testing/datatest_fixtures.py

```diff
@@ -33,11 +33,12 @@ def download_ser_data(request, processor_props, ranked_data_path, experiment, py
     """
     Get the binary ICON data from a remote server.
 
-    Session scoped fixture which is a prerequisite of all the other fixtures in this file.
+    Fixture which is a prerequisite of all the other fixtures in this file.
     """
     try:
-        if not request.config.getoption("datatest"):
-            pytest.skip("not running datatest marked test")
+        # we don't want to run this ever if we are not running datatests
```

Comment:
Not sure this is correct: My reasoning was: if we run with `--datatest-skip` we set the "not datatest" in the config, and then we don't need to download data, so return and skip the body.

Reply:
I think it makes sense, but wouldn't it be simpler to avoid the `try-except` block and provide a falsy value as default to `request.config.getoption()`:
```python
if "not datatest" in request.config.getoption("-k", ""):
    return
```